fe/fe-core/src/main/java/com/starrocks/staros/StarOSBDBJEJournalSystem.java

```diff
@@ -185,6 +187,7 @@ public void replayTo(long journalId) throws StarException {
             Util.stdoutWithTime(e.getMessage());
             System.exit(-1);
         } catch (Exception e) {
+            LOG.warn("got exception when replay star mgr journal", e);
```

Comment:
it is a replay code path, no need to print the detailed stack trace.

Reply:
When I encountered this issue, there was little useful information in fe.warn.log, with only a null present. This is not conducive to troubleshooting the problem.

Reply:
Additionally, due to the fact that StarException in `throw new StarException(ExceptionCode.JOURNAL, e.getMessage());` does not support passing the `e` type as a parameter, this leads to incomplete stack trace information. This is also the reason for the proactive invocation of `LOG.warn`.